themes/bootstrap3/templates/_ui/components/confirm-menu.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+  $toggleLabel = $this->transEsc($this->label ?? '{->label}');
+  $toggleClass = trim('confirm__toggle ' . $this->escapeHtmlAttr(($this->toggleClass ?? 'btn')));
+
+  $desc = $this->transEsc($this->description ?? 'Are you sure?');
+
+  $actionType = $this->action->type ?? 'submit';
+
+  $confirmLabel = $this->transEsc($this->confirm ?? 'confirm_dialog_yes');
+  $cancelLabel = $this->transEsc($this->cancel ?? 'confirm_dialog_no');
+?>
+
+<div class="confirm-menu">
+  <button class="<?=$toggleClass ?>"><?=$toggleLabel ?></button>
+
+  <div class="confirm__options">
+    <p class="confirm__desc"><?=$desc ?></p>
+
+    <ul role="menu" class="confirm__menu">
+      <li role="none">
+        <?php if ($actionType == 'submit'): ?>
+          <button role="menuitem" type="submit" class="confirm__confirm">
+            <?=$this->icon('ui-success', 'confirm__icon') ?>
+            <?=$confirmLabel ?>
+          </button>
+        <?php elseif ($actionType == 'link'): ?>
+          <a role="menuitem" class="confirm__confirm" href="<?=$this->escapeHtmlAttr($this->action->value) ?>">
+            <?=$this->icon('ui-success', 'confirm__icon') ?>
+            <?=$confirmLabel ?>
+          </a>
+        <?php endif; ?>
```

Comment:
Should we have an else clause for undefined action types? (e.g. throw an exception)